packages/core/src/codewhispererChat/tools/fsWrite.ts

```diff
@@ -73,24 +77,14 @@ export class FsWrite {
     }
 
     public async queueDescription(updates: Writable): Promise<void> {
-        const sanitizedPath = sanitizePath(this.params.path)
-        const changes = await this.getDiffChanges()
-        const language = await getLanguageForFilePath(sanitizedPath)
-
-        const diff = getDiffMarkdown(changes, language)
-        updates.write(diff)
+        // Write an empty string because FsWrite should only show a chat message with header
```

Comment:
Don't fully follow here, why do we need an empty string?

Reply:
This is just to make sure we create a chat message for this tool so that we can add a header to it